backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/view/IView.java

```diff
@@ -208,6 +208,12 @@ default boolean hasAttributesSupport()
 
 	<T> List<T> retrieveModelsByIds(DocumentIdsSelection rowIds, Class<T> modelClass);
 
+	@NonNull
+	default <T> Stream<T> streamModelsByIds(@NonNull final DocumentIdsSelection rowIds, @NonNull final Class<T> modelClass)
+	{
+		return retrieveModelsByIds(rowIds, modelClass).stream();
+	}
+
```

Comment:
I'm thinking it would be better to actually have impls everywhere, instead of using a default impl here, as generally, we should not encourage `loadAll`  and then `stream`.. even though for the other views it makes sense... wdyt @metas-ts ? (only reason I'm hesitating it's the budget :) )